packages/block-editor/src/components/block-list/block-selection-button.js

```diff
@@ -223,6 +223,12 @@ function BlockSelectionButton( { clientId, rootClientId, blockElement } ) {
 
 				if ( navigateDown ) {
 					nextTabbable = focus.tabbable.findNext( blockElement );
+
+					if ( ! nextTabbable ) {
+						nextTabbable =
+							blockElement.ownerDocument.defaultView.frameElement;
+						nextTabbable = focus.tabbable.findNext( nextTabbable );
+					}
```

Comment:
Is this the writing flow behavior? Did it move to the selection button? and what does this fallback do?

Reply:
It's navigation mode behaviour. The fallback is for when you're in an iframe and there is no next element to tab to. I should probably find a more elegant solution for this.